mcp-worker/src/ably.ts

```diff
@@ -0,0 +1,39 @@
+import Ably from 'ably/build/ably-webworker.min'
+import type { DevCycleJWTClaims } from './types'
+
+export async function publishMCPInstallEvent(
+    env: { ABLY_API_KEY?: string },
+    claims: DevCycleJWTClaims,
+): Promise<void> {
+    if (!env.ABLY_API_KEY) {
+        throw new Error('ABLY_API_KEY is required to publish MCP events')
+    }
+    if (!claims.org_id) {
+        throw new Error('org_id is required in claims to publish MCP events')
+    }
+
+    const channel = `${claims.org_id}-mcp-install`
+
+    try {
+        const ably = new Ably.Rest.Promise({ key: env.ABLY_API_KEY })
```

Comment:
This should be a different API key than our others

Reply:
It is already